Mage.Sets/src/mage/cards/b/BlatantThievery.java

```diff
@@ -29,7 +23,7 @@ public BlatantThievery(UUID ownerId, CardSetInfo setInfo) {
         this.getSpellAbility().addEffect(new GainControlTargetEffect(Duration.Custom, true)
                 .setTargetPointer(new EachTargetPointer())
                 .setText("for each opponent, gain control of target permanent that player controls"));
-        this.getSpellAbility().setTargetAdjuster(BlatantThieveryAdjuster.instance);
+        this.getSpellAbility().setTargetAdjuster(new EachOpponentPermanentTargetsAdjuster(new TargetPermanent()));
```

Comment:
perhaps rather than a new TargetAdjuster in the constructor, declare it `private static final TargetAdjuster`, like filters? unless there's a reason not to.

Reply:
I guess that's fine but it seems more complicated. Is the benefit from ensuring less memory duplication worth the extra lines of code and separation between the target and the ability it's targeting for?

Reply:
it's no worse than it was before with the custom enum, and it's barely any added code. but I agree there is also some benefit to the way you have it, and the memory impact is unlikely to matter.